### PR TITLE
Change "World Wind" to "WorldWind"

### PR DESCRIPTION
**Pull request by eirizarry**
### Description of the Change
- All instances of "World Wind" have been changed to the single world project name "WorldWind". This includes all instances in the source code comments and other supporting documentation within the WorldWind Android repository. 

### Why Should This Be In Core?, Benefits
- This pull request is necessary to make the WorldWind project name consistent throughout all of our repositories. It is also part of an epic which updates WorldWind Android's documentation.

Closes [#140](https://github.com/NASAWorldWind/WorldWindAndroid/issues/140), closes [#144](https://github.com/NASAWorldWind/WorldWindAndroid/issues/144). 

### Potential Drawbacks
- Number of files modified may add time to the review process. 

### Applicable Issues

### CODE REVIEW

File: .idea/runConfigurations/worldwind_androidTest.xml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <component name="ProjectRunConfigurationManager">
```
Comment (pdavidc) [inline .idea/runConfigurations/worldwind_androidTest.xml]: Change needed: Any idea why this Android Studio config file has changed?

File: worldwind/build.gradle [hunk 2]
```diff
@@ -94,7 +94,7 @@ bintray {
         name = System.getenv('BINTRAY_PACKAGE')
         version {
             name = android.defaultConfig.versionName
-            desc = 'World Wind Android v' + android.defaultConfig.versionName
+            desc = 'WorldWind Android v' + android.defaultConfig.versionName
```
Comment (pdavidc) [inline worldwind/build.gradle]: Note: Safe code change. This affects the Bintray package name.

File: worldwind/build.gradle [hunk 3]
```diff
@@ -123,8 +123,8 @@ task compileJavadocSources(type: Javadoc) {
     description 'Creates the Javadoc API documentation'
     source android.sourceSets.main.java.srcDirs
     include('**/*.java')
-    title 'NASA World Wind Android'
-    options.windowTitle 'NASA World Wind Android'
+    title 'NASA WorldWind Android'
```
Comment (pdavidc) [inline worldwind/build.gradle]: Note: These JavaDoc arguments affect the documentation displayed at https://worldwind.arc.nasa.gov/android/docs/

File: worldwind/src/main/java/gov/nasa/worldwind/WorldWindow.java [hunk 4]
```diff
@@ -188,8 +188,8 @@ protected void init(EGLConfigChooser configChooser) {
         this.setRenderer(this);
         this.setRenderMode(GLSurfaceView.RENDERMODE_WHEN_DIRTY); // must be called after setRenderer
 
-        // Log a message with some basic information about the world window's configuration.
-        Logger.log(Logger.INFO, "World Window initialized");
+        // Log a message with some basic information about the WorldWindow's configuration.
+        Logger.log(Logger.INFO, "WorldWindow initialized");
```
Comment (pdavidc) [inline worldwind/src/main/java/gov/nasa/worldwind/WorldWindow.java]: Note: Safe code change. This affects log output.

File: worldwind/src/main/java/gov/nasa/worldwind/util/Logger.java [hunk 5]
```diff
@@ -97,7 +97,7 @@
         messageTable.put("missingViewport", "The viewport is null");
         messageTable.put("missingVector", "The vector is null");
         messageTable.put("missingVersion", "The version is null");
-        messageTable.put("missingWorldWindow", "The world window is null");
+        messageTable.put("missingWorldWindow", "The WorldWindow is null");
```
Comment (pdavidc) [inline worldwind/src/main/java/gov/nasa/worldwind/util/Logger.java]: Note: Safe code change. This affects log output.

File: worldwind/src/main/java/gov/nasa/worldwind/util/TaskService.java [hunk 6]
```diff
@@ -40,7 +40,7 @@ protected ExecutorService executorService() {
     }
 
     protected ThreadFactory threadFactory() {
-        final String threadName = "World Wind Task Service ";
+        final String threadName = "WorldWind Task Service ";
```
Comment (pdavidc) [inline worldwind/src/main/java/gov/nasa/worldwind/util/TaskService.java]: Note: Safe code change. This affects thread group names.

File: README.md [hunk 7]
```diff
@@ -32,7 +32,7 @@ Official World Wind Android releases have the latest stable features, enhancemen
 
 ## License
 
-    NASA WORLD WIND
+    NASA WorldWind
```
Comment (pdavidc) [inline README.md]: Change needed: Let's keep the uppercase WORLDWIND found in license files.

File: .idea/runConfigurations/worldwind_androidTest.xml (hunk 1 above)
Comment (eirizarry) [reply .idea/runConfigurations/worldwind_androidTest.xml]: I don't know why this changed. I will revert it back.

**Pull request merged**